pyodide-build/pyodide_build/_f2c_fixes.py

```diff
@@ -487,7 +487,14 @@ def scipy_fix_cfile(path: str) -> None:
 
     for lib in ["lapack", "blas"]:
         if path.endswith(f"cython_{lib}.c"):
-            header_path = Path(path).with_name(f"_{lib}_subroutines.h")
+            header_name = f"_{lib}_subroutines.h"
```

Comment:
Maybe put this in a separate `find_header` function?

Reply:
Or maybe this is fine as is...

Reply:
I moved it to a separate function. Thanks for the suggestion!